.github/workflows/test.yml

```diff
@@ -38,3 +33,27 @@ jobs:
       - name: Run tests
         run: ${SAGE} runtests.py -v
 
+  test:
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        sage-version:
+          - '9.3'
+          - '9.4'
+          - '9.5'
+          - '9.6'
+    steps:
+      - name: Install sage
+        run: |
+          conda config --add channels conda-forge
+          conda install sage=${{ matrix.sage-version }} -y
```

Comment:
You can use mamba for faster solves

          conda config --add channels conda-forge
          conda install mamba -y
          mamba install sage=${{ matrix.sage-version }} -y

Reply:
Thanks very much for the tip, I would not have guessed that would make such a big difference! #204